packages/react/src/Bento/BentoItem.features.stories.tsx

```diff
@@ -0,0 +1,480 @@
+import React from 'react'
+import {StoryFn, Meta} from '@storybook/react'
+import {INITIAL_VIEWPORTS} from '@storybook/addon-viewport'
+import {Bento} from '.'
+import {Text, Link, ColorModesEnum} from '../'
+import placeholderImage from '../fixtures/images/placeholder-600x400.png'
+
+export default {
+  title: 'Components/Bento/Item',
+  component: Bento,
+  parameters: {
+    layout: 'fullscreen',
+    viewport: {
+      viewports: INITIAL_VIEWPORTS,
+    },
+  },
+} as Meta<typeof Bento>
+
+export const HeadingWithEmphasizedText: StoryFn<typeof Bento> = () => (
+  <Bento.Item>
+    <Bento.Content>
+      <Bento.Heading as="h3">
+        <em>Heading</em> with emphasis
+      </Bento.Heading>
+      <Link href="#">Call to action</Link>
+    </Bento.Content>
+    <Bento.Visual position="50% 100%">
+      <img className="test" alt="placeholder, blank area with an gray background color" src={placeholderImage} />
+    </Bento.Visual>
+  </Bento.Item>
+)
+
+HeadingWithEmphasizedText.parameters = {
+  viewport: {
+    defaultViewport: 'iphonexr',
+  },
+}
+
+export const VisualPositionBottom: StoryFn<typeof Bento> = () => (
+  <Bento.Item>
+    <Bento.Content>
+      <Bento.Heading as="h3">Heading</Bento.Heading>
+      <Text>
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit. In sapien sit ullamcorper id. Aliquam luctus sed turpis
+        felis nam pulvinar risus elementum.
+      </Text>
+      <Link href="#">Call to action</Link>
+    </Bento.Content>
+    <Bento.Visual position="50% 100%">
+      <img className="test" alt="placeholder, blank area with an gray background color" src={placeholderImage} />
+    </Bento.Visual>
+  </Bento.Item>
+)
+
+VisualPositionBottom.parameters = {
+  viewport: {
+    defaultViewport: 'iphonexr',
+  },
+}
+
+export const FlowColumn: StoryFn<typeof Bento> = () => (
+  <Bento.Item flow="column">
+    <Bento.Content>
+      <Bento.Heading as="h3">Heading</Bento.Heading>
```

Comment:
Similar comment to the test class, we're setting `as="h3"` manually in a lot of the stories and docs examples, can we remove those please and rely on the default value? It may give the impression that users need to set this each time.

```suggestion
      <Bento.Heading>Heading</Bento.Heading>
```

Reply:
This has been updated in the following [SHA](https://github.com/primer/brand/pull/390/commits/12573536c648bb891828e7aa42347f12b44fb46a)